All tests are passing

gabebw opened the pull request:
The tests were failing before for a few reasons:

* Haml was required, but not in the Gemfile
* A few views used `-` instead of `=`, so nothing was being output
* Something in the pipeline (Rouge, maybe?) now outputs a `<code>` tag in addition to the `<pre>` tag

Arcovion commented:
Thank you, these were long standing bugs.
I think the docs need to be updated for the `code` helper re: `-` &rarr; `=`, do you know if the `erb` example still works or if it needs adjusting `<% %>` &rarr; `<%= %>`?

gabebw commented:
Oh, weird. I thought that `-` vs `=` in Haml was a mistake, but it looks like the `code` helper is supposed to work without `=` in Haml/erb, per the docs.

It does work without `<%=`, just `<%`, in ERb - what if I change it and middleman-syntax just requires `=` everywhere, in Haml and erb?

(I have not taken a very hard look at the middleman-syntax/rouge/haml pipeline, so I'm not sure exactly where the bug is in Haml.)

Arcovion commented:
Yea it's strange, Padrino helpers are now using `=` in Haml and `<% %>` in ERb, so this actually matches that behaviour... it's inconsistent but they say it's not easy to fix.
I think using `=` for the helper is best in Haml/Slim, but for ERb we probably want to match the other helpers.

https://github.com/padrino/padrino-framework/issues/1839
cc: @tdreyno

gabebw commented:
Thanks for the pointer to the issue! I've updated the documentation to reflect Haml/erb differences, and also updated the HTML structure in the docs because Rouge generates `<pre><code>` now, not just `<div><pre>`.

Arcovion commented:
Nice, looks good to me!

tdreyno commented:
Thanks @gabebw

gabebw commented:
You're welcome! It's a great project.
